dts/bindings/dma/gd,gd32-dma.yaml

```diff
@@ -4,6 +4,32 @@
 description: |
   GD32 DMA controller
 
+  The node has 3-cells.
+
+  channel: Select channel for data transmitting
+  priority: Set priority for this channel
+  peripheral: Select peripheral to connect DMA (Only for GD32F4xx series)
```

Comment:
Can `peripheral` not be determined by the device node that has the `dmas` property?

Reply:
Exactly as you pointed out, it can determine which peripheral to use in the client code that use the DMA.
And it doesn't have to be configurable.
I removed it.

Reply:
I think it can't determine.

The peripheral assignment is tied to the DMA channel.
For example, from  [GD32f450 manual](https://www.gigadevice.com/manual/gd32f450xxxx-user-manual/) Page 211 Table 10-6,
SPI0-RX can use with pair of channel-0 and peripheral-3 or channel-2 and peripheral-3.
Channel is already configurable with the property.
Peripheral is also needs to be configurable with property to follow channel configuration.

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/48756#issuecomment-1209025306

I have changed design since this discussion.
So I assume this Change Request is no longer needed. Could you check it please?